parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -4481,7 +4491,7 @@ static void jdf_generate_startup_hook( const jdf_t *jdf )
             "    parsec_task_class_t* tc = (parsec_task_class_t*)__parsec_tp->super.super.task_classes_array[i];\n"
             "    __parsec_chore_t* chores = (__parsec_chore_t*)tc->incarnations;\n"
             "    uint32_t idx = 0, j;\n"
-            "    for( j = 0; NULL != chores[j].hook; j++ ) {\n"
+            "    for( j = 0; PARSEC_DEV_NONE != chores[j].type; j++ ) {\n"
```

Comment:
why ? It seems that the two lead to similar outcome, but I question the need for the change.